snowpack/assets/hmr-client.js

```diff
@@ -160,11 +160,11 @@ async function runJsModuleAccept({url: id, bubbled}) {
     return false;
   }
   const acceptCallbacks = state.acceptCallbacks;
-  const updateID = Date.now();
   for (const {deps, callback: acceptCallback} of acceptCallbacks) {
     const [module, ...depModules] = await Promise.all([
-      import(id + `?mtime=${updateID}`),
-      ...deps.map((d) => import(d + `?mtime=${updateID}`)),
+      import(id + `?mtime=${updateId}`),
+      // FIXME we can't know the correct updateId for dependencies at this point
+      ...deps.map((d) => import(d + `?mtime=${updateId}`)),
```

Comment:
This is still a problem that needs solving... At this point, we have no guarantee that the `updateId` we use here matches the one known by the server (that would be used in subsequent import rewriting, and could result in duplicated version of the dep).